movie/src/components/Icons/DotIcon/DotIcon.css

```diff
@@ -1,21 +1,3 @@
-.more-detail {
-  background-color: rgb(234, 244, 244);
-  width: 30px;
-  height: 30px;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  border-radius: 30px;
-  margin: 5px;
-  right: 2px;
-  position: absolute;
-}
-
-.more-detail:hover {
-  cursor: pointer;
-  background-color: rgb(151, 237, 237);
-}
-
 body:not(.modal-open){
```

Comment:
its still `.modal-open`? why not `.modalOpen`

Reply:
It is not className bong, it is default class of open modal. It used to make default padding-right: 15px to padding-right:0px.